### ISSUE TITLE
Scaling issue, causing incorrect supposed applet popup position

### ISSUE THREAD

**Issue by Jacob-Vlijm** [solus-project/budgie-desktop#1883]
budgie-desktop 10.5 on Ubuntu Budgie 19.10

![second](https://user-images.githubusercontent.com/8932980/66551689-3f14a100-eb48-11e9-8200-6f739f343a30.png)

When using scaling, opening (e.g.) Quicknote, and subsequently clicking on the popup, makes it disappear as if we clicked _outside_ the popup. If we look at the x/y position of the mouse click, multiplying it with the scaling factor makes it _stay_, as if we click inside the applets area. 

All is best explained by the image above: scaling is set to 400% (to make it clear). Clicking inside the applet's popup makes it go away, if we click at both x and y of 400% the popup's center position, it is experienced as a click on the popup. 

Seems like we need a fix for the difference in scaling handling between Wnck and Gtk/Gdk,

**Comment by Jacob-Vlijm** [comment]
Yep, saw those, no one mentioned the off-position of the applet' s click rectangle however. should be an easy fix though.

**Comment by JoshStrobl** [comment]
Sorry I meant #1387, been a long day :D

**Comment by Jacob-Vlijm** [comment]
Ah, of course :)
